easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-intel-2015b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'Doxygen'
+version = '1.8.10'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('CMake', '3.4.1'),
```

Comment:
because in a later PR where we also need Doxygen, you'll have the same problem again, doesn't matter